editions/tw5.com/tiddlers/system/doc-styles.tid

```diff
@@ -285,6 +285,14 @@ ol.doc-github-contributors li {
 	overflow: hidden;
 	text-overflow: ellipsis;
 }
+.doc-op-comparison {
+	table-layout: fixed;
+	width: 80%;
+}
+.doc-op-comparison th .doc-operator {
```

Comment:
I think `.doc-op-comparison` classes should **not** be part of this PR. They are too usecase specific. 

There is an existing `tc-max-width-80` which sets the max-width for every container. It's defined near the [Table Utility Classes](https://github.com/Jermolene/TiddlyWiki5/blob/f4626aa69e39fb439cad674aea596ab7fc5b15c4/themes/tiddlywiki/vanilla/base.tid#L328) in the base styles.

The `table-layout: fixed;` **is new**. So there should be a new table specific utility class eg: `tc-table-layout-fixed` that's not a pretty name but it explains itself. 

The docs for Table Utility classes is at: https://tiddlywiki5-8tfwqvccw-jermolene.vercel.app/#Tables%20in%20WikiText%20CSS%20Utility%20Classes

Reply:
The new and needed classes may be assigned to the table itself. But there should be a separated PR with the new table-class. Once it is merged, the docs will be fixed automagically.

Reply:
In CSS, `max-width: 80%` is not the same as `width: 80%`, so I could not use `tc-max-width-80`.
Creating more table utility classes may seem like a good idea, but I feel there will be endless discussions about the loss of utility due to an over-abundance of classes. These docs should not depend on another far-future PR to look right.
I could just define these styles inline if additional doc-styles are unwelcome. But then the next comparison table by someone else will look different (which might be the case anyway) or code will be needlessly duplicated.